content/geoip/updating-databases.mdx

```diff
@@ -129,6 +129,23 @@ servers can make HTTPS connections to the following hostname:
 
 - `mm-prod-geoip-databases.a2649acb697e2c09b632799562c076f2.r2.cloudflarestorage.com`
 
+For example, using the permalink for the GeoIP2 City CSV database, you can issue
+a `curl` command like the following:
```

Comment:
I wonder if this would make more sense after (4) below. @christophermluna, thoughts?